NEW Add separate frontend bundle and TextFilter component

robbieaverill opened the pull request:
Resolves #74

ScopeyNZ commented on client/src/components/Frontend/TextFilter.js:
```diff
@@ -0,0 +1,90 @@
+    return i18n.inject(
+      i18n._t(
+        'CKANTextFilter.FILTER_BY_MANY',
+        'Search by {firstColumns} or {lastColumn}'
```
Oxford comma?

ScopeyNZ commented on client/src/components/Frontend/tests/TextFilter-story.js:
```diff
@@ -0,0 +1,43 @@
+        TextFieldComponent={TextField}
+      />
+    </div>
+  ));
```
❤️

robbieaverill commented:
**Hold** until we've clarified the design aspect

robbieaverill commented:
I've updated this to use a provided label via props instead of a generated placeholder on the input field

ScopeyNZ commented on client/src/components/Frontend/TextFilter.js:
```diff
@@ -0,0 +1,58 @@
+
+export { TextFilter as Component };
+
+export default inject(
```
Do we want to use Injector on the front end components? Considering our advice will be "just copy the code and edit for customisations" I feel that keeping it simple is best.

robbieaverill replied:
Good point - I didn't even think about it. I guess we should just render inputs directly instead

ScopeyNZ replied:
I guess there's nothing to prevent us from having Reactstrap as a dependency though.

robbieaverill commented:
Updated - I've also removed the "Frontend" separation of components to match a bit closer with #122. Whichever gets merged first will need to update the webpack config to match the other.

robbieaverill commented:
Just realised that the story book doesn't make sense now that we've removed dynamic label generation - have updated it

robbieaverill commented:
Replaced in #135
